### ISSUE TITLE
Incorrect roster height

### ISSUE THREAD

**Issue by floriancargoet** [jcbrand/converse.js#360]
When one configures converse so that the `add-converse-contact` box is not there, there's an empty space under the contacts list.

This is due to the CSS:
```
  #converse-roster {
    height: calc(100% - 68px);
  }
```

The 68 pixels are for both the "status box" and the "add contact box" but it should be dynamic.

**Comment by jcbrand** [comment]
I'm not sure how this can be fixed generically. The CSS cannot be aware of what configuration settings are sent to converse.js.

**Comment by floriancargoet** [comment]
The JS could add a CSS class or set the style directly on the element.
```
if (/* add-contact-box is hidden */) {
    this.$roster.addClass('full-height');
    // or
    this.$roster.addClass('no-add-contact');
    // or
    this.$roster.css({
        'max-height' : '100%'
    });
}
```

**Comment by jcbrand** [comment]
Didn't think of that :)

Fixed.

**Comment by floriancargoet** [comment]
I just realised that `this.$roster` wasn't the correct component but I guess you corrected that.

Other possible solution, CSS only: the adjacent selector. I've not tested it but it should be possible.
It's not easily extensible, doesn't work if the add-contact is after... I prefer the JS solution.
```
#converse-roster {
  height: height when the roster is alone;
}
.add-converse-contact + #converse-roster {
  height: height when the add-converse-contact box is there;
}
```

**Comment by jcbrand** [comment]
Look at the commit connected to this issue and you'll see what I did.

**Comment by jcbrand** [comment]
BTW, I just released version 0.9.2 which contains your fixes.

**Comment by floriancargoet** [comment]
Great!
